api/routes.py

```diff
@@ -26,6 +27,14 @@
 from db.schemas.section import SECTION_SCHEMA_MAP
 
 
+def is_valid_uuid(value):
+    try:
+        obj = uuid.UUID(value, version=4)
```

Comment:
I don't think it's worth enforcing version 4 here - let it be any valid UUID. Maybe we decide to change from version 4 in the future, and we shouldn't have to worry about this check.

Reply:
Updated